debian/control

```diff
@@ -50,6 +50,7 @@ Build-Depends: bc,
                python (>= 2.7),
                python-all-dev,
                python-nose,
+	       python-prettytable,
```

Comment:
debian packaging does not run "make check", so we can remove `python-prettytable` from Build-Depends.